admin/bin/qserv-parallel-cmd.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python
+
+# LSST Data Management System
+# Copyright 2014 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+
+"""
+User-friendly tool allowing to manage a set of Qserv nodes via ssh
```

Comment:
You are saying "User-friendly" in so many places, I think I'd drop it, they should all be user-friendly and we don't have to say it :)